src/super_gradients/training/Computer_Vision_Models_Pretrained_Checkpoints.md

```diff
@@ -68,12 +68,17 @@ All the available models are listed in the column `Model name`.
 | PP-LiteSeg T75        | pp_lite_t_seg75   | Cityscapes | 768x1536   | 77.56 | **5.20ms**              | **32.28ms**                          |                    **38.03ms**                    |
 | DDRNet 23 slim        | ddrnet_23_slim    | Cityscapes | 1024x2048  | 78.01 | **5.74ms**              | **32.01ms**                          |                    **45.18ms**                    |
 | DDRNet 23             | ddrnet_23         | Cityscapes | 1024x2048  | 80.26 | **12.74ms**             | **39.01ms**                          |                   **106.26ms**                    |
+| DDRNet 39             | ddrnet_39         | Cityscapes | 1024x2048  | 81.32 | **23.57ms**             | **52.41ms**                          |                   **145.79ms**                    |
 | STDC 1-Seg50          | stdc1_seg50       | Cityscapes | 512x1024   | 75.11 | **3.34ms**              | **30.12ms**                          |                    **27.54ms**                    |
 | STDC 1-Seg75          | stdc1_seg75       | Cityscapes | 768x1536   | 77.8  | **5.53ms**              | **32.490ms**                         |                     **43.88**                     |
 | STDC 2-Seg50          | stdc2_seg50       | Cityscapes | 512x1024   | 76.44 | **4.12ms**              | **30.94ms**                          |                    **32.03ms**                    |
 | STDC 2-Seg75          | stdc2_seg75       | Cityscapes | 768x1536   | 78.93 | **6.95ms**              | **33.89ms**                          |                   **54.48ms**                     |
 | RegSeg (exp48)        | regseg48          | Cityscapes | 1024x2048  | 78.15 | **12.03ms**             | **38.91ms**                          |                    **78.20ms**                    |
+| DDRNet 23 slim - CWD  | ddrnet_23_slim    | Cityscapes | 1024x2048  | 79.41 | **5.74ms**              | **32.01ms**                          |                    **45.18ms**                    |
```

Comment:
We can just keep the best checkpoints and simply state in the recipe that the ious were reached with the KD ones.

Reply:
Done [update model-zoo](https://github.com/Deci-AI/super-gradients/pull/731/commits/87192ff439dd9afbde5bea1607025f7360f7d6e0)